src/main/c/Makefile

```diff
@@ -11,17 +11,20 @@ endif
 no_cexts: clean
 	@echo "WARNING: TRUFFLERUBY_CEXT_ENABLED is set to false - not building cexts" 1>&2
 
-all: $(ROOT)/lib/cext/ruby.su $(ROOT)/lib/mri/openssl.su $(ROOT)/lib/mri/zlib.su $(ROOT)/lib/mri/syslog.su
+all: $(ROOT)/lib/cext/trufflemock.bc $(ROOT)/lib/cext/ruby.su $(ROOT)/lib/mri/openssl.su $(ROOT)/lib/mri/zlib.su $(ROOT)/lib/mri/syslog.su
```

Comment:
One nicer way to do this is to express in the dependencies below.
I think we should make `cext/Makefile:` depend on `trufflemock.bc`.